### PR TITLE
desktop Portfolio design completed.

### PR DESCRIPTION
**Pull request by DilsherB**
Desktop Portfolio design completed.

### CODE REVIEW

**Comment by raezon** [comment]
## Status: Pending   ♻️ ♻️ 

**Hi @DilsherB,**

You did a really great job!  unfortunetly you must enhance your footer

télécharger


# Add border radius to ther top-left corner of your footer 
check your figma you are missing a border radius on the corner top left 

image


## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by henry-dura — APPROVED** [review]
#Good job!
**Highlights**

- No linters errors
- followed project requirements
- Good Github flow

**Review by alicemirigo92 — APPROVED** [review]
You have done a great job adding the changes requested by a previous reviewer 🎯, and there is nothing else left to do other than it's time to merge it 🚀🚀.

To Highlight:
Correct user of GitHub flow ✔
Descriptive PR summary ✔
STATUS: APPROVED ✅✅✅

**Comment by DilsherB** [comment]
I am thankful to all of you for your reviews.

**Review by TadesseDev — COMMENTED** [review]
### Hi @DilsherB 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑🛑
yPkfDFf

You have submitted a project that has linter tests falling. Please make sure all linters are set up and passing before your next review request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -252,3 +252,73 @@ textarea {
   padding: 12px;
   font-size: 17px;
 }
+
+@media only screen and (min-width: 768px) {
+  #header {
+    padding: 20px 150px;
+  }
+
+  #headline {
```
Comment (Olamarx) [inline style.css:279]: - [ ] The nav under this id should not display the hamburger on the desktop version, you can do that by adding display none to the nav id.

image

File: index.html [hunk 2]
```diff
@@ -30,112 +30,133 @@ <h1 id="introHeader">I'am Dilsher Balouch, Glad to see you!</h1>
...
+                <h2 id="introHeader">About Myself</h2>
+                <p id="intro">I’m a software developer! I can help you build a product, 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+                </p>
+                <div id="sociHeading">LET'S CONNECT</div>
+                <ul id="sociConnector">
+                    <li><a href="https://twitter.com/_brilliantMindz"><img src="images/twitter.svg"></a></li>
+                    <li><a href="https://www.linkedin.com/in/dilsher-balouch-7139432a/"><img src="images/linkedin.svg"></a></li>
```
Comment (Olamarx) [inline index.html]: - [ ] Kindly notice that the two blocks in this section should not be of equal width, the languages, framework and skills block should be wider than the other.

### Figma
image

### Your project
image

Comment (Olamarx) [inline index.html]: - [ ] Kindly notice that the two blocks in this section should not be of equal width, the languages, framework and skills block should be wider than the other.

### Figma
image

### Your project
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @DilsherB,

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linters passed ✔️ 
- Media query ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @DilsherB , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

200w


## Highlights
All linters are passing :heavy_check_mark: 
Neat and well organized code :heavy_check_mark: 


## Optional suggestions

- [ ] Kindly note that it is never a good practice to use one branch for many pull request. Please open branch for different pull request once it has been approved. :pray: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**